servers/dsub/jobs/test/test_jobs_controller_google.py

```diff
@@ -115,6 +115,15 @@ def test_query_jobs_by_start(self):
         self.wait_status(self.api_job_id(job), ApiStatus.RUNNING)
         self.assert_query_matches(QueryJobsRequest(start=date), [job])
 
+    def test_aggregtion_jobs_invalid_project_id(self):
+        time_frame = 'DAYS_7'
+        project_id = 'should-be-an-invalid-id'
+        resp = self.client.open(
+            '/aggregations?projectId={}&timeFrame={}'.format(
+                project_id, time_frame),
+            method='GET')
+        self.assert_status(resp, 400)
```

Comment:
A 400 is fine if the requestor doesn't provide a projectId. But this test passes a project ID that doesn't exist. It should be expecting a 404 or 403 (see other tests).